modules/arena.py

```diff
@@ -0,0 +1,256 @@
+"""Arena."""
+from modules.graphics import Circle
+import abc
+from typing import List, Callable
+import time
+import numpy as np
+from numpy.linalg import norm
+from modules.player import Player
+from settings import PhysicsConsts
+
+
+class ArenaLayer(abc.ABC):
+    """
+    Abstract class representing the physical fields.
+
+    The fields are associated with the board.
+    """
+
+    @abc.abstractmethod
+    def force(
+        self,
+        player: Player,
+        input_force: np.ndarray,
+        total_force: np.ndarray,
+        max_input: float,
+        dt: float,
+    ) -> np.ndarray:
+        """Return force for specific player."""
+
+
+class FrictionLayer(ArenaLayer):
+    """Representing the physical friction field."""
+
+    def __init__(self, friction_matrix: np.ndarray,
+                 friction_const: Callable) -> None:
```

Comment:
`Callable` can be more strictly typed by specifying the types of input args and return args.
This way: `Callable[[inputType1, inputType2, ...], OutputType]`.